texastribune/pops
# Fix broken recover_list template

#### What's this PR do?
Gets the recover_list template working again at /admin/stories/story/recover/. Also bumps the version twice and gets this app synced with what's on our pypi.

#### Why are we doing this? How does it help us?
While prepping for the django upgrade, I found that "Recover deleted stories" was broken due to a deprecated url template syntax (fixed [here](https://github.com/texastribune/pops/pull/4/files#diff-306ef261805dc2d719db7e56b2b13750R8)). I also found that the version of pops that we have been using (0.2.0tt2) was not on GitHub.

Now reporters/editors can find and recover deleted stories, and tech can find the latest pops code.

#### How should this be manually tested?

Since it couldn't hurt (right? *right??*), I've already deployed this to pypi as 0.2.0tt3, and pointed the `django-upgrade-prep` branch at the new version. This way we shouldn't have to deal with the pain of editable installs again.

From the texastribune app:

- `git checkout django-upgrade-prep && git pull origin django-upgrade-prep && pip install -Ur requirements/base.txt`
- Find a story in the admin, set its publication status to "Trash", then delete it.
- Navigate to /admin/stories/story/recover/ and make sure you see it and can recover it.

#### Technical debt note
Someday we will remove pops, but in the meantime it's good to have the most recent version back on GitHub.
